client/web/src/tracking/eventLogger.ts

```diff
@@ -82,6 +82,9 @@ export class EventLogger implements TelemetryService {
             secure: true,
             // We only read the cookie with JS so we don't need to send it cross-site nor on initial page requests.
             sameSite: 'Strict',
+            // Specify the Domain attribute to ensure subdomains (about.sourcegraph.com) can receive this cookie.
+            // https://developer.mozilla.org/en-US/docs/Web/HTTP/Cookies#define_where_cookies_are_sent
+            domain: 'sourcegraph.com',
```

Comment:
Can we use `location.hostname` so this could be tested locally when using `sourcegraph.test`? (note: not `host`, because Cookies are not distinguished by port)
```suggestion
            domain: location.hostname,
```

Reply:
👍🏾  just pushed this up in a new commit